src/components/Header/TopNav/Dropdown.js

```diff
@@ -59,6 +59,10 @@ const DropdownNameWrapper = styled.span`
 const DropdownName = styled.span`
   padding-right: ${remcalc(6)};
   outline: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
```

Comment:
styled components handles vendor prefixing, you should only need `user-select: none` here

Reply:
cunning!